docs/modules/ROOT/pages/auth/subscriptions.adoc

```diff
@@ -0,0 +1,29 @@
+[[subscriptions]]
+= Subscriptions
+
+xref::subscriptions/index.adoc[Subscriptions] can be used along with `@auth`, however, some operations are not supported. To setup rules,
+use the `SUBSCRIBE` operation.
+
+```graphql
+type Movie {
+    title: String!
+}
+
+extend type Movie @auth(rules: [{ isAuthenticated: true, operations: [SUBSCRIBE] }])
+```
+
+== Authentication
+If the authentication rules `isAuthenticated` and `allowUnauthenticated` are not met, the subscription request will fail and no events will
+be sent.
+
+== Roles
+Roles can be set for subscriptions. Only requests matching the roles set will be accepted.
```

Comment:
Is it worth clarifying that these rules are checked on the initial request to create the subscription?